### ISSUE TITLE
Webpack error: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'

### ISSUE THREAD

**Issue by binary64** [MunifTanjim/node-bitbucket#61]
I get many of these errors (I'm on Typescript 3.9.3 and nextjs latest)

```
Creating an optimized production build

Failed to compile.

C:/p/myapp/node_modules/bitbucket/src/plugins/auth/types.ts
ERROR in C:/p/myapp/node_modules/bitbucket/src/plugins/auth/types.ts(1,10):
1:10 Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.
  > 1 | export { Options } from '../../client/types'
      |          ^
    2 | export { RequestOptions } from '../../endpoint/types'
    3 | 
    4 | export type APIClient = import('../../client/types').APIClient


> Build error occurred
Error: > Build failed because of webpack errors
```

It seems the solution is to change those "export" to "export type" throughout your codebase.

**Comment by MunifTanjim** [comment]
@binary64 Fix should be published with [`v2.2.0`](https://www.npmjs.com/package/bitbucket/v/2.2.0) :tada:

**Comment by binary64** [comment]
You nailed it. Thanks!
